### PR TITLE
OC-6203

### PR DESCRIPTION
**Pull request by rooney**
Add a link in OpenClinica to access participate

### CODE REVIEW

File: web/src/main/java/org/akaza/openclinica/control/submit/ListStudySubjectTableFactory.java [hunk 1]
```diff
@@ -1334,4 +1365,47 @@ private String formatDate(Date date) {
...
+
+    public void setOcuiParameter() {
+        String host = null;
+        String currentAddress = servletRequest.getRequestURL().toString().split("OpenClinica")[0];
+        String tmpAddress = null;
+        try {
+            Class.forName("org.postgresql.Driver");
+
+            /* need to be modified based on database location */
+            connection = DriverManager.getConnection("jdbc:postgresql://127.0.0.1:5432/epro", "epro","epro");
```
Comment (rooney) [inline web/src/main/java/org/akaza/openclinica/control/submit/ListStudySubjectTableFactory.java]: @kkrumlian is this correct? or should I add a new entry in datainfo.properties for this? (say let's call it portalDatabaseURL)

File: web/src/main/resources/datainfo.properties [hunk 2]
```diff
@@ -330,5 +330,5 @@ ldap.userData.organization=company
 Participant Portal
 ####################################################################
 # To connect to Participant Portal , use the below URL
-portalURL =
+portalURL=http://host2.ocui.com:8081
```
Comment (kkrumlian) [inline web/src/main/resources/datainfo.properties]: Revert this

File: web/src/main/java/org/akaza/openclinica/web/pmanage/ParticipantPortalRegistrar.java [hunk 3]
```diff
@@ -145,9 +145,32 @@ public String getStudyHost(String studyOid)
         if (json.isEmpty()) return "";
         
     	JSONObject studyHost = json.getJSONObject("study");
-		
+    	String[] arrayUrl = pManageUrl.split(":");
```
Comment (kkrumlian) [inline web/src/main/java/org/akaza/openclinica/web/pmanage/ParticipantPortalRegistrar.java]: Put this code in its own method

File: web/src/main/resources/datainfo.properties (hunk 2 above)
Comment (rooney) [inline web/src/main/resources/datainfo.properties]: @kkrumlian OK reverted on https://github.com/rooney/OpenClinica/commit/2a0b76c0994ff490505a4ae053b29a5560e4be9b

File: web/src/main/java/org/akaza/openclinica/web/pmanage/ParticipantPortalRegistrar.java (hunk 3 above)
Comment (rooney) [inline web/src/main/java/org/akaza/openclinica/web/pmanage/ParticipantPortalRegistrar.java]: @kkrumlian OK refactored on https://github.com/rooney/OpenClinica/commit/2a0b76c0994ff490505a4ae053b29a5560e4be9b

**Comment by kkrumlian** [comment]
:+1: Looks good

**Pull request merged**